.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,68 @@
+<!-- Describe the intention of the changes being proposed. What problem does it solve or functionality does it add? -->
+
+- ...
+
+## Does this introduce a breaking change?
```

Comment:
I think this is a good question to ask, while the rest below are things that are not that important at the moment and only increase contribution friction. How about we keep this question and drop the rest?